### DISCUSSION TITLE
Bar plot

### DISCUSSION THREAD

**Discussion by dinaf39** [teamalgoritma/community#659] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/130343206/fa5e6d42-e264-4071-b367-84d101b63efe)
![image](https://github.com/teamalgoritma/community/assets/130343206/2c7a6260-60ea-4c61-8113-3e1f64b60b40)
kenapa bar plotnya kosong dan tidak ada data nya ya

**Comment by rusdipermana** [comment, marked as answer]
![cmiw](https://github.com/teamalgoritma/community/assets/6158006/9d45a4a5-c8cc-4901-8838-809da7042a7c)

semoga membantu

**Reply by dinaf39** [reply]
![image](https://github.com/teamalgoritma/community/assets/130343206/227e41dd-ad22-4500-a244-4e67179a05aa)
tapi kenapa pada sumbu x hasilnya msih berbeda dengan contoh ya?
![image](https://github.com/teamalgoritma/community/assets/130343206/9ef46435-786e-4359-9685-a106ec222956)
atau memang berbeda

**Reply by rusdipermana** [reply]
Untuk kasusnya silakan disesuaikan ya bu dengan soalnya, yg saya berikan mungkin berbeda. Silakan dicoba🙂

**Comment by nadyadtm** [comment]
“Halo @dinaf39 , untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah lebih dari 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.”
